.circleci/config.yml

```diff
@@ -20,6 +20,37 @@ executors:
         environment:
           POSTGRES_USER: user
           POSTGRES_DB: ccq_test
+  integration-executor:
```

Comment:
could we use "e2e" throughout, instead of "integration" for these tests? - see https://dsdmoj.atlassian.net/wiki/spaces/EPT/pages/4402872687/ADR11+E2E+tests